scripts/maintenance/computational-clusters/osparc_clusters.py

```diff
@@ -77,11 +74,50 @@ class DynamicInstance(AutoscaledInstance):
 MINUTE: Final[int] = 60
 HOUR: Final[int] = 60 * MINUTE
 
-COMPUTATIONAL_INSTANCE_NAME_PARSER: Final[parse.Parser] = parse.compile(
-    r"osparc-computational-cluster-{role}-{swarm_stack_name}-user_id:{user_id:d}-wallet_id:{wallet_id:d}"
-)
+DEFAULT_COMPUTATIONAL_EC2_FORMAT: Final[
+    str
+] = r"osparc-computational-cluster-{role}-{swarm_stack_name}-user_id:{user_id:d}-wallet_id:{wallet_id:d}"
+DEFAULT_DYNAMIC_EC2_FORMAT: Final[str] = r"osparc-dynamic-autoscaled-worker-{key_name}"
 
 DEPLOY_SSH_KEY_PARSER: Final[parse.Parser] = parse.compile(r"osparc-{random_name}.pem")
```

Comment:
not sure if it is always like this, with the specific syntax...

Reply:
this is modified depending on the ENVs